etc/kayobe/environments/ci-multinode/README.md

```diff
@@ -77,19 +77,6 @@ cd ../stackhpc-kayobe-config
 source kayobe-env --environment ci-multinode
 ```
 
-6. Add hooks for `configure-vxlan.yml` and `growroot.yml`
```

Comment:
The hooks should not be committed to the repository - these instructions should stay

Reply:
These instructions should stay.